### DISCUSSION TITLE
Problem with registering an account on Firefly III

### DISCUSSION THREAD

**Discussion by SuperDarius-git** [firefly-iii/firefly-iii#6696] [Firefly III]
Hi guys, I have asked the following question on Reddit r/selfhosted: https://www.reddit.com/r/selfhosted/comments/z5dgkj/problem_registering_logging_into_firefly_iii_with/

I then got an answer from James, but I am still struggling with the same problem, which is: Registering/logging in initially after installing on Docker. When I click on "Register", it just does nothing.

Thanks
Darius

**Comment by JC5** [comment]
Is there an error in the browser console? Ctrl-shift-i

**Comment by SuperDarius-git** [comment]
This is all it gives after pressing button once.: Content Security Policy: `The page’s settings blocked the loading of a resource at http://money.super.org.za/register (“form-action”). 2`
This is what shows when I refresh the page: ```
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
Content Security Policy: Ignoring “'self'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'self'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https://www.google-analytics.com/analytics.js” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “https://www.google-analytics.com/analytics.js” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring source “strict-dynamic” (Only supported within script-src).
Content Security Policy: Ignoring source “strict-dynamic” (Only supported within script-src).
Content Security Policy: Couldn’t parse invalid host 'strict-dynamic'
Content Security Policy: Ignoring “'self'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'self'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring “'unsafe-inline'” within script-src: ‘strict-dynamic’ specified
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
Content Security Policy: Ignoring source “strict-dynamic” (Only supported within script-src).
Content Security Policy: Ignoring source “strict-dynamic” (Only supported within script-src).
Content Security Policy: Couldn’t parse invalid host 'strict-dynamic' 3
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. register
Layout was forced before the page was fully loaded. If stylesheets are not yet loaded this may cause a flash of unstyled content. jquery.js:2:50784
WebGL context was lost. 3
InstallTrigger is deprecated and will be removed in the future. content.js:1:7021
Content Security Policy: The page’s settings blocked the loading of a resource at http://money.super.org.za/ (“base-uri”). 2
Password fields present in a form with an insecure (http://) form action. This is a security risk that allows user login credentials to be stolen. register
function iSkysoft_ISAllmytube_Plugin()
Keepvid.Main.user.js:224:13
object, embed, video, iframe, a, undefined 2 Keepvid.Main.user.js:262:17
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://b65243d3-1557-4b9e-9973-37da5b6ca3af/assets/fonts/Roboto-Bold.ttf (“font-src”). 2
Content Security Policy: The page’s settings blocked the loading of a resource at chrome-extension://b65243d3-1557-4b9e-9973-37da5b6ca3af/assets/fonts/Roboto-Regular.ttf (“font-src”). 2
Error: An unexpected error occurred spoofer.js:1:38935
Error: JSON.parse: unexpected character at line 1 column 1 of the JSON data undefined
Object { 0: HTMLDocument https://money.super.org.za/register, length: 1 }
contentscript.js:2687:17
<empty string> contentscript.js:2755:19
Password fields present in a form with an insecure (http://) form action. This is a security risk that allows user login credentials to be stolen. register
Error: Could not establish connection. Receiving end does not exist. content.js:9:40302

```
I don't understand a word.

Thank you
Darius

**Reply by JC5** [reply]
It looks like the config values you edited did not help. Are you using nginx or some kind of proxy manager?

**Reply by SuperDarius-git** [reply]
Yes, I am using NGINX Proxy Manager...

**Comment by SuperDarius-git** [comment]
I solved i t with some help with someone on Reddit. There were a few parameters from the .env file incorrect. Thank you for your help in any way...

Darius
